collect_app/src/test/java/org/odk/collect/android/entities/EntitiesRepositoryTest.kt

```diff
@@ -57,7 +57,7 @@ abstract class EntitiesRepositoryTest {
     }
 
     @Test
-    fun `#save updates existing entity with matching id in different list`() {
+    fun `#save creates entity with matching id in different list`() {
```

Comment:
This behaviour isn't something we have an opinion as far as I'm aware - we don't expect this to happen in real life.